libs/payments/paypal/src/lib/paypal.manager.ts

```diff
@@ -35,6 +37,19 @@ export class PayPalManager {
     await this.client.baUpdate({ billingAgreementId, cancel: true });
   }
 
+  /**
+   * Create billing agreement using the ExpressCheckout token.
+   *
+   * If the call to PayPal fails, a PayPalClientError will be thrown.
+   *
+   */
+  async createBillingAgreement(
+    options: CreateBillingAgreementOptions
```

Comment:
This should take a string with token as an argument so that it matches your jsdoc